tests/integration_tests/scheduler/mock_torque_bin/qsub

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
Suggest to have these mocked queue binaries in a common directory,  `mock_queue_bin` or something

Reply:
Changed to just `bin`